src/FSharp.Data.Adaptive/AdaptiveIndexList/AdaptiveIndexList.fsi

```diff
@@ -51,6 +51,22 @@ module AList =
     /// Creates an alist using the given reader-creator.
     val ofReader : create : (unit -> #IOpReader<IndexListDelta<'T>>) -> alist<'T>
     
+    /// Creates an alist using the given value generator
+    val constant: value: (unit -> IndexList<'T>) -> alist<'T>
```

Comment:
I made this public so we can inline creations of `AList` (the inlined implementation uses it) - is the name ok?  It's perhaps a bit counterintuitive - maybe `AList.constantDelayed` or `AList.ofIndexListDelayed`

Reply:
I think the name is just okay, the type and the comment make it clear what happens...